packages/connect/e2e/README.md

```diff
@@ -12,7 +12,6 @@ you may use the following params:
 
 ```
 -f <semver string such as 2.5.2>
--u <url of unix-frozen-debug-build, such as https://gitlab.com/satoshilabs/trezor/trezor-firmware/-/jobs/2730055101/artifacts/file/core/build/unix/trezor-emu-core>
```

Comment:
btw this would be a great feature only if it had a predicatble way to provide this arg